### PR TITLE
Fixes #11058: 4.0 reaches EOL

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/11058

### CODE REVIEW

File: release-data/rudder-4.0.cson [hunk 1]
```diff
@@ -6,7 +6,7 @@ release = "4.0"
 release-status = "final"
 
 # Is this version currently supported?
-supported = true
+supported = false
```
Comment (peckpeck) [inline release-data/rudder-4.0.cson]: Shouldn't we just remove the file ?

**Comment by amousset** [comment]
Commit modified

File: release-data/changelogs/changelog-4.0.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Change logs for Rudder 4.0.\* versions
 
-Rudder 4.0 is currently a maintained release of Rudder, with
-an End of Life planned the 30th June 2017.
+Rudder 3.0 has reached End of Life the 30th June 2017.
```
Comment (fanf) [inline release-data/changelogs/changelog-4.0.md]: 4.0

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
@fanf fixed

**Pull request merged**